beam/core/src/main/java/cz/o2/proxima/beam/direct/io/BatchLogRead.java

```diff
@@ -153,7 +153,7 @@ public ProcessContinuation process(
         PartitionList restriction = Objects.requireNonNull(tracker.currentRestriction());
         Partition part = Objects.requireNonNull(restriction.getFirstPartition());
 
-        BlockingQueueLogObserver observer =
+        final BlockingQueueLogObserver.BatchLog observer =
```

Comment:
suggest `BlockingQueueLogObserver.Batch` or `BlockingQueueLogObserver.BatchLogObserver`, the `BatchLog` looks a little weird.

Reply:
👍